### PR TITLE
make env variable name consistent

### PR DESCRIPTION
**Pull request by chengchingwen**
as mentioned in #101

### CODE REVIEW

**Comment by oxinabox** [comment]
technically this is a breaking change, without deprecation, but i doubt anyone is using that setting

**Comment by oxinabox** [comment]
hmm I don't know why this is breaking, somewhere it is trying to convert `Inf` to `Int`.
But not sure why.
As `Inf` is a valid input

**Comment by chengchingwen** [comment]
Looks like it's a bug caused by `Dates.Millisecond` call by `HTTP.download` where it do `Millisecond(1000*update_period)`.

**Comment by chengchingwen** [comment]
@oxinabox I make a [PR](https://github.com/JuliaWeb/HTTP.jl/pull/439)  to fix it, but we might need wait for that pr get merged and released. Anything we can do before that?

**Comment by oxinabox** [comment]
I have merge rights on HTTP.jl so I will make sure this gets in fairly quick

**Comment by chengchingwen** [comment]
Do we need deprecated warning for `DATADEP_ PROGRESS_UPDATE_PERIOD`?

**Comment by oxinabox** [comment]
ideally, yes.

**Comment by oxinabox** [comment]
Cool 0.8.5 of HTTP.jl has been released. So need to up that lower bound and make a patch-bump to version in Project.toml

File: Project.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 
 [compat]
 ExpectationStubs = "0.2.0"
-HTTP = "0.7.0, 0.8"
+HTTP = ">= 0.8.5"
```
Comment (oxinabox) [inline Project.toml]:
```suggestion
HTTP = "0.8.5"
```

Wr stick to semver bounds as I don't like my code just breaking.
And I don't like having to retroactively edit the registry to stop it downgrading to the last version without bounds when it does break

**Comment by chengchingwen** [comment]
bump

**Comment by oxinabox** [comment]
thanks

**Pull request merged**